example/data/collections/L2_HR_PIXC-000.json

```diff
@@ -5,7 +5,8 @@
   "provider_path": "ignored-provider-path/test-data",
   "granuleId": "^.*$",
   "granuleIdExtraction": "^(.*)(\\.h5|\\.cmr)",
-  "sampleFileName": "L2_HR_PIXC_product_0001-of-4154.h5",  
+  "duplicateHandling": "replace",
```

Comment:
@laurenfrederick - I added the `duplicateHandling` for all the test collections to `replace` so that they don't fallback to the `error` behavior

Reply:
👍